docs/guidelines.md

```diff
@@ -100,7 +101,7 @@ the `link` function, the `[]` notation, instead of the `.` notation, must be
 used when accessing these properties. See the example below.
 
 ```js
-module.directive('goDirectiveExample',
+module.directive('ngeoExampleDirective',
```

Comment:
We never use `Directive` in the proper name of the directive (but only in init function name).